libcni/api_test.go

```diff
@@ -497,6 +514,25 @@ var _ = Describe("Invoking plugins", func() {
 					})
 				})
 
+				Context("no cached config just result", func() {
```

Comment:
@mccv1r0 can we change the context here to be more inline with Ginkgo style? Eg combine all the It(), Context(), etc messages together. For this test you have:

Context("when CHECK is called with a configuration version"
Context("no cached config just result"
It("passes a prevResult to the plugin

So that would be "when CHECK is called with a configuration version no cached config just result it passes a prevResult to the plugin" and that doesn't parse in English. Compare to the immediately previous:

Context("when CHECK is called with a configuration version"
Context("less than 0.4.0"
It("fails as CHECK is not supported before 0.4.0"

"when CHECK is called with a configuration version less than 0.4.0 it fails as CHECK is not supported before 0.4.0" which does parse.